components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -11,29 +11,29 @@
     <ng-container>
       <form id="ruleForm" [formGroup]="ruleForm" (ngSubmit)="saveRule()">
 
-        <chef-form-field class="display3">
+        <chef-form-field id="create-name">
           <label>
-            <span class="label">Rule Name</span>
-            <chef-input id="ruleName" ngDefaultControl formControlName="name"></chef-input>
+            <span class="label">Rule Name <span aria-hidden="true">*</span></span>
+            <chef-input ngDefaultControl formControlName="name"></chef-input>
           </label>
           <chef-error
-            *ngIf="ruleForm.get('name').hasError('required') && ruleForm.get('name').dirty"
-          >Name is required</chef-error>
+            *ngIf="(ruleForm.get('name').hasError('required') || ruleForm.get('name').hasError('pattern')) && ruleForm.get('name').dirty"
+          >Rule Name is required</chef-error>
         </chef-form-field>
 
-        <chef-form-field class="display3">
+        <chef-form-field id="create-type">
           <label>
-            <span class="label">Resource Type</span>
-            <select id="resourceType" ngDefaultControl formControlName="type">
-              <option value='NODE'>Node</option>
-              <option value='EVENT'>Event</option>
+            <span class="label">Resource Type <span aria-hidden="true">*</span></span>
+            <select id="create-type-dropdown" ngDefaultControl formControlName="type">
+              <option value='node'>Node</option>
+              <option value='event'>Event</option>
             </select>
           </label>
           <chef-error
-            *ngIf="ruleForm.get('type').hasError('required') && ruleForm.get('type').dirty"
+            *ngIf="ruleForm.get('type').hasError('required') && ruleForm.get('type').touched"
```

Comment:
We use `dirty` on plain inputs, but it turns out that `touched` is much better for `<select>` elements. That way, tabbing through it without selecting a value activates the error display.